public/js/clients/firefox.js

```diff
@@ -101,9 +101,8 @@ function initPage(actions) {
   threadClient = getThreadClient();
 
   setupCommands({ threadClient, tabTarget, debuggerClient });
-
   tabTarget.on("will-navigate", actions.willNavigate);
-  tabTarget.on("navigate", actions.navigate);
+  tabTarget.on("navigate", actions.willNavigate);
```

Comment:
I opted against changing navigate after reading the comments in that method. But it makes sense, I'll move it it to navigate.